packages/style-value-parser/src/at-queries/__tests__/media-query-transform-test.js

```diff
@@ -184,22 +164,18 @@ describe('Media Query Transformer', () => {
 
   test('single word condition', () => {
     const originalStyles = stylex.create({
-      colorMode: {
-        mode: {
-          default: 'normal',
-          '@media (color)': 'colorful',
-          '@media (monochrome)': 'grayscale',
-        },
+      mode: {
+        default: 'normal',
+        '@media (color)': 'colorful',
+        '@media (monochrome)': 'grayscale',
       },
     });
 
     const expectedStyles = {
-      colorMode: {
-        mode: {
-          default: 'normal',
-          '@media (color) and (not (monochrome))': 'colorful',
-          '@media (monochrome)': 'grayscale',
-        },
+      mode: {
+        default: 'normal',
+        '@media (color) and (not (monochrome))': 'colorful',
```

Comment:
Not a blocker for this PR, but note that this should be simplified in the future. color and monochrome should be considered mutually exclusive.